rero_ils/modules/serializers.py

```diff
@@ -122,6 +122,41 @@ def complete_bucket_with_attribute(results, bucket_name, resource_cls,
                     if attr in resource:
                         term[attr] = resource.get(attr)
 
+    @staticmethod
+    def enrich_bucket_with_data(results, bucket_name, search_cls,
+                                attributes_name):
+        """Complete a bucket by adding new keys based on resource attributes.
+
+        :param results: the ES results data (containing aggregations).
+        :param bucket_name: the bucket name to perform.
+        :param search_cls: the related search class.
+        :param attributes_name: attributes to load from search data.
+        """
+        attributes_name = attributes_name or []
```

Comment:
this can be done with a default value in params, no?

Reply:
@jma no. You explained a couple month ago that use empty array as default function arguments isn't a good idea because it's only compute once, when the code is 'compile' ;-)

Reply:
I think such as @zannkukai.
https://docs.python.org/3/reference/compound_stmts.html#:~:text=Default%20parameter%20values%20are,the%20function%2C%20e.g.%3A
but maybe, we are wrong

Reply:
@zannkukai you are right, but in this case it should be ok as the default value is a constant. The problem happens when for example you want to set the default value as the current date.